### PR TITLE
Workspace facets

### PR DESCRIPTION
**Pull request by Hellblazer**

### CODE REVIEW

File: animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspaceImporter.java [hunk 1]
```diff
@@ -178,141 +177,190 @@ public void setScope(WorkspaceScope scope) {
         this.scope = scope;
     }
 
-    private void classifyAgencyAttributes() {
-        for (ClassifiedAttributeContext classified : wsp.getAgencyAttributeClassifications()) {
+    /**
+     * @param facet
+     */
+    private void agencyAuthorizations(FacetContext facet) {
+        // TODO Auto-generated method stub
```
Comment (meleemistress) [inline animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspaceImporter.java]: Did you mean to leave this blank?

Comment (Hellblazer) [inline animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspaceImporter.java]: Yes.  I still have to add the cross existential entity network authorizations, and I wanted to leave a TODO so I could remember to do it.

File: animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspaceImporter.java [hunk 2]
```diff
@@ -917,6 +787,39 @@ private void loadUnits() {
         }
     }
 
+    /**
+     * @param facet
+     */
+    private void locationAuthorizations(FacetContext facet) {
+        // TODO Auto-generated method stub
```
Comment (meleemistress) [inline animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspaceImporter.java]: ?

File: animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspacePresentation.java [hunk 3]
```diff
@@ -366,25 +286,14 @@ public WorkspacePresentation(WorkspaceContext context) {
...
-    public List<NetworkAuthorizationContext> getStatusCodeNetworkAuthorizations() {
-        if (context.statusCodes == null
-            || context.statusCodes.networkAuthorizations() == null) {
-            return Collections.emptyList();
-        }
-
-        List<NetworkAuthorizationContext> ruleforms = context.statusCodes.networkAuthorizations().networkAuthorization();
-
-        return ruleforms == null ? Collections.emptyList() : ruleforms;
+        return context.statusCodes.facets().facet();
```
Comment (meleemistress) [inline animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspacePresentation.java]: So... does this mean there will only ever be one status code facet?

Comment (meleemistress) [inline animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspacePresentation.java]: Never mind. I understand now

**Pull request merged**